src/backend/marsha/core/lti/utils.py

```diff
@@ -9,6 +9,57 @@ class PortabilityError(Exception):
     """An error raised when trying to access a resource that is not portable."""
 
 
+def get_selectable_resources(model, lti):
+    """Filter resources available to a LTI select request.
+
+    This function is generic and will use the `model` argument to filter resources.
+
+    Parameters
+    ----------
+    lti : Type[LTI]
+
+    model:
+        The model we want to get.
+
+    Returns
+    -------
+    A queryset of available model instances to the LTI request
+
+    """
+    if not (lti.is_instructor or lti.is_admin):
+        return model.objects.none()
+
+    consumer_site = lti.get_consumer_site()
+
+    playlist_reachable_from = Playlist.objects.filter(
+        portable_to__lti_id=lti.context_id,
+        portable_to__consumer_site_id=consumer_site.id,
+    )
+
+    return model.objects.select_related("playlist").filter(
+        # Q() if (lti.is_instructor or lti.is_admin) else model.get_ready_clause(),
```

Comment:
We should maybe keep it as a safe guard.

Reply:
I did a similar check at line 29

Reply:
Then remove this line